test/connectivity.test.js

```diff
@@ -126,6 +131,18 @@ describe('Connectivity Checker for BrowserStack Components', function () {
         errorMessage: null,
         result: "Passed",
         statusCode: 301
+      },{
+        data: '{"data":"value"}',
+        description: "HTTPS Request To " + constants.HUB_STATUS_URL.replace("http://", "https://") + " With Proxy",
```

Comment:
shall we have separate URLs for http and https instead of replacing at every occurrence? for both rails and hub.... will be cleaner...

Reply:
in test helper functions http is replaced by https. Doing same changes in those helper function will take more revamp.

Reply:
can be resolved. I think this is done...